projects/ngx-prx-styleguide-storybook/src/app/app.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'sb-root',
+  template: `
+    <!--The content below is only a placeholder and can be replaced.-->
```

Comment:
This looks like a leftover from generating the Storybook.

Reply:
Not really. Storybook needs a base application project to initialize from. Since the ngx-prx-styleguide project is a library, it couldn't be used as a base. This component is defaults from using ng cli to generate an application. The only thing I know that is pulled into storybook is the project's styles as defined in angular.json. Not sure how much of the base angular app can be gutted. I'll add some instructions to this in case someone runs it.